### PR TITLE
Remove Put from FabricLocation which was removed from the product ove…

### PR DESCRIPTION
**Pull request by cednation**
…r a year ago

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/FabricLocation.json [hunk 1]
```diff
@@ -93,40 +93,6 @@
                     "nextLinkName": "nextLink"
                 },
                 "x-ms-odata": "#/definitions/FabricLocation"
-            },
-            "put": {
-                "x-ms-examples": {
-                    "Creates or updates a fabric location.  This will fail if called outside deployment.": {
-                        "$ref": "./examples/FabricLocation/Put.json"
```
Comment (nschonni) [inline specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/FabricLocation.json]: The example file will need to be deleted to make Avocado happy

**Comment by adxsdknet** [comment]
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/6015 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/6015/commits/502b4f51437875f10598f2a52ae01bd0ceacc0ae

File: specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/FabricLocation.json [hunk 2]
```diff
@@ -93,40 +93,6 @@
...
-                        "$ref": "Fabric.json#/parameters/SubscriptionIdParameter"
-                    },
-                    {
-                        "$ref": "Fabric.json#/parameters/ResourceGroupParameter"
-                    },
-                    {
-                        "$ref": "Fabric.json#/parameters/ApiVersionParameter"
-                    },
-                    {
-                        "$ref": "#/parameters/FabricLocationObjectParameter"
```
Comment (martabarillas) [inline specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/FabricLocation.json]: Remove the local parameter not longer been used.
#/parameters/FabricLocationObjectParameter

File: specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/FabricLocation.json (hunk 1 above)
Comment (sarangan12) [reply specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/FabricLocation.json]: Agreed with @nschonni.

@cednation Please remove the example file too. 

Also, Please be aware this is a breaking change and thus needs ARM review

**Review by sarangan12 — CHANGES_REQUESTED** [review]

File: specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/FabricLocation.json (hunk 1 above)
Comment (cednation) [reply specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/FabricLocation.json]: done

File: specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/FabricLocation.json (hunk 2 above)
Comment (cednation) [reply specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/FabricLocation.json]: done

**Review by martabarillas — APPROVED** [review]

**Review by sarangan12 — APPROVED** [review]

**Pull request merged**